modules/admin/con-rhdh-operator-deployment-config.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assembly:
+// assembly-rhdh-operator-deployment-config
+
+[id="con-rhdh-operator-deployment-config_{context}"]
+= {product} Operator deployment configuration
```

Comment:
```suggestion
= {product} Deployment configuration
```

IMO, this can be confusing as it can be understood as configuring the Operator itself, which is not the case here. This is mostly about how to configure various aspects of the RHDH [Deployment](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/) (in the sense of a Kubernetes Deployment resource) that will be created by the Operator when a user [deploys RHDH](https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.2/html/installing_red_hat_developer_hub_on_openshift_container_platform/assembly-install-rhdh-ocp#proc-install-rhdh-ocp-operator_assembly-install-rhdh-ocp-operator).